source/base/Profiler.ooc

```diff
@@ -65,6 +70,8 @@ Profiler: class {
 	}
 	resetAll: static func { This _profilers apply(func (profiler: This) { profiler reset() }) }
 	dispose: static func {
+		while (This _profilers count)
```

Comment:
I can change this, but code like `if (pointer)` is currently used almost everywhere.
Do you remember any argument against it ?  I'm just curious